sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -163,3 +163,21 @@ export class ApplicationSummaryAPIOutDTO {
   submitted?: Date;
   status: ApplicationStatus;
 }
+
+/**
+ * History of SIN validations associated with a user.
+ */
+export class SINValidationsAPIOutDTO {
+  id: number;
+  sin: string;
+  createdAt: Date;
+  isValidSIN?: boolean;
+  sinStatus?: string;
+  validSINCheck?: string;
+  validBirthdateCheck?: string;
+  validFirstNameCheck?: string;
+  validLastNameCheck?: string;
+  validGenderCheck?: string;
```

Comment:
In the data table I saw response code. Which property of DTO is populating it? 
![image](https://user-images.githubusercontent.com/54600590/175354861-2b86c91f-06d1-4779-aced-db11f045f4b1.png)

Reply:
![image](https://user-images.githubusercontent.com/61259237/175358379-c7336825-3b26-46f8-9124-1a6746a714ec.png)

Reply:
Ah, sinStatus it is. 👍